vernac/vernacinterp.mli

```diff
@@ -9,12 +9,25 @@
 (** Interpretation of extended vernac phrases. *)
 
 type deprecation = bool
+
 type vernac_command = Genarg.raw_generic_argument list -> unit -> unit
+(** For each "rule"
+    (see the [rule] non-teriminal defined in [grammar/vernacextend.mlp] file)
+    camlp5 generates one function of the [vernac_command] type which
+    defines how a given Vernacular command, introduced by the "rule",
+    should be interpreted.
+
+    The actual "arguments" of the Vernacular command
+    (see the [args] non-terminal defined in [grammar/vernacextend.mlp] file)
+    are passed via the first parameter of this function. *)
```

Comment:
To avoid too much cross-references, I would not talk about `rule` and simply say:

(* This function registers the interpretation function associated to `VERNAC EXTEND` block of a given name. *)